Parsers/ASimDns/ARM/ASimDns/README.md

```diff
@@ -1,15 +1,15 @@
-# Infoblox NIOS ASIM DNS Normalization Parser
+# Source agnostic ASIM Dns Normalization Parser
 
-This template deploys the ASIM DNS schema parser for Infoblox NIOS. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM Source agnostic Dns schema parser. The parser is a part of the Azure Sentinel Information Model.
```

Comment:
Should be DNS with all capital. Also, any clue why the PR identify the origin (Infoblox)?

Reply:
For some reason it compares wrong files. The diff did not cope well with the file renames